includes/connect/class-wc-stripe-connect-rest-controller.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'WC_Stripe_Connect_REST_Controller' ) ) {
+	/**
+	 * Stripe Connect base REST controller class.
+	 */
+	class WC_Stripe_Connect_REST_Controller extends WP_REST_Controller {
```

Comment:
Should this class be abstract? e.g. rest_base only appears to be defined in the descendants below

Reply:
Made this abstract in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1267/commits/de0d448f1d4901e53adc21927403ef9ec0ab0f1b